content/warnings/dont-call-proptypes.md

```diff
@@ -1,50 +1,50 @@
 ---
-title: Don't Call PropTypes Warning
+title: Avertissement Don't Call PropTypes
 layout: single
 permalink: warnings/dont-call-proptypes.html
 ---
 
-> Note:
+> Note :
 >
-> `React.PropTypes` has moved into a different package since React v15.5. Please use [the `prop-types` library instead](https://www.npmjs.com/package/prop-types).
+> `React.PropTypes` a été déplacé dans un paquet différent depuis React v15.5. Merci d'utiliser [la bibliothèque `prop-types` à la place](https://www.npmjs.com/package/prop-types).
 >
->We provide [a codemod script](/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes) to automate the conversion.
+>Nous fournissons [un script codemod](/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes) pour automatiser la conversion.
 
-In a future major release of React, the code that implements PropType validation functions will be stripped in production. Once this happens, any code that calls these functions manually (that isn't stripped in production) will throw an error.
+Dans une future version majeure de React, le code implémentant les fonctions de validation PropType sera supprimé en production. Une fois que cela se produit, tout code appelant ces fonctions manuellement (qui n'est pas supprimé en production) générera une erreur.
 
-### Declaring PropTypes is still fine {#declaring-proptypes-is-still-fine}
+### Déclarer des PropTypes est toujours correct {#declaring-proptypes-is-still-fine}
 
-The normal usage of PropTypes is still supported:
+L'utilisation normale de PropTypes est toujours prise en charge :
 
 ```javascript
 Button.propTypes = {
   highlighted: PropTypes.bool
 };
 ```
 
-Nothing changes here.
+Rien ne change ici.
 
-### Don’t call PropTypes directly {#dont-call-proptypes-directly}
+### N'appelez pas PropTypes directement {#dont-call-proptypes-directly}
 
-Using PropTypes in any other way than annotating React components with them is no longer supported:
+L'utilisation de PropTypes autrement que par l'annotation de composants React n'est plus soutenu :
```

Comment:
```suggestion
L'utilisation de PropTypes autrement que par l'annotation de composants React n'est plus acceptée :
```

« Soutenu » ?!  Ce n'est pas “support” comme dans “support group” ou “looking for support”, mais comme “support a feature / use case”, donc « prise en charge » (le franglais « supporter » étant un barbarisme, puisque le véritable sens français de « supporter » est soit « servir de support physique » soit « accepter / tolérer » quelque chose de désagréable).

Ici toutefois il s'agit moins d'une prise en charge (comme « prise en charge native par les navigateurs ») que d'une acceptation, car on parle d'une mauvaise pratique dès le départ, recourir à « accepté(e) » (par React) est plus fort dans ce contexte.